.env.template

```diff
@@ -1,6 +1,13 @@
-THOTH_CEPH_BUCKET=
-THOTH_CEPH_BUCKET_PREFIX=
+# source ceph
+THOTH_DEPLOYMENT_NAME="ocp-test"
+
+THOTH_S3_ENDPOINT_URL="https://s3.upshift.redhat.com/
+THOTH_CEPH_BUCKET="thoth"
+THOTH_CEPH_BUCKET_PREFIX="data"
 THOTH_CEPH_KEY_ID=
 THOTH_CEPH_SECRET_KEY=
-THOTH_DEPLOYMENT_NAME=
-THOTH_S3_ENDPOINT_URL=
+
+# dest ceph
+THOTH_DOCUMENT_SYNC_DST="s3://thoth/test"
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
```

Comment:
We should add something like 
```suggestion
AWS_SECRET_ACCESS_KEY=

# Job specifics
THOTH_DOCUMENT_SYNC_CONFIGURED_SOLVERS=<something>
```
In order to test solver sync also, not just the rest.

Reply:
Like we could simply use the value from the stage environment, somehting like that.